From: =?utf-8?q?Florian_M=C3=BCllner?= <fmuellner@gnome.org>
Date: Tue, 12 Jan 2021 17:31:15 +0100
Subject: weather: Provide application-id and contact-info

Those are now required in order to use any online providers.

https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/3577

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/1565>

Origin: https://gitlab.gnome.org/GNOME/gnome-shell/-/commit/d5ca1bc62f
---
 js/misc/weather.js | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/js/misc/weather.js b/js/misc/weather.js
index 0c293bf..a8b8f6c 100644
--- a/js/misc/weather.js
+++ b/js/misc/weather.js
@@ -72,7 +72,11 @@ var WeatherClient = class {
             GWeather.Provider.METAR |
             GWeather.Provider.MET_NO |
             GWeather.Provider.OWM;
-        this._weatherInfo = new GWeather.Info({ enabled_providers: providers });
+        this._weatherInfo = new GWeather.Info({
+            enabled_providers: providers,
+            application_id: 'org.gnome.Shell',
+            contact_info: 'https://gitlab.gnome.org/GNOME/gnome-shell/-/raw/master/gnome-shell.doap',
+        });
         this._weatherInfo.connect_after('updated', () => {
             this._lastUpdate = GLib.DateTime.new_now_local();
             this.emit('changed');
